[core]/es_extended/client/modules/adjustments.lua

```diff
@@ -41,7 +41,86 @@ function Adjustments:HealthRegeneration()
     end
 end
 
-function Adjustments:AmmoAndVehicleRewards()
+function Adjustments:PointLoop()
+    CreateThread(function()
+        while self.isPointing do
+            local camPitch = GetGameplayCamRelativePitch()
+            local camHeading = GetGameplayCamRelativeHeading()
+            local cosCamHeading = Cos(camHeading)
+            local sinCamHeading = Sin(camHeading)
+
+            camPitch = math.max(-70.0, math.min(42.0, camPitch))
+            camPitch = (camPitch + 70.0) / 112.0
+            camHeading = math.max(-180.0, math.min(180.0, camHeading))
+            camHeading = (camHeading + 180.0) / 360.0
+
+            local coords = GetOffsetFromEntityInWorldCoords(ESX.PlayerData.ped, (cosCamHeading * -0.2) - (sinCamHeading * (0.4 * camHeading + 0.3)), (sinCamHeading * -0.2) + (cosCamHeading * (0.4 * camHeading + 0.3)), 0.6)
+            local ray = StartShapeTestCapsule(coords.x, coords.y, coords.z - 0.2, coords.x, coords.y, coords.z + 0.2, 0.4, 95, ESX.PlayerData.ped, 7)
+            local _, blocked = GetRaycastResult(ray)
+
+            SetTaskMoveNetworkSignalFloat(ESX.PlayerData.ped, "Pitch", camPitch)
+            SetTaskMoveNetworkSignalFloat(ESX.PlayerData.ped, "Heading", camHeading * -1.0 + 1.0)
+            SetTaskMoveNetworkSignalBool(ESX.PlayerData.ped, "isBlocked", blocked)
+            SetTaskMoveNetworkSignalBool(ESX.PlayerData.ped, "isFirstPerson", GetCamViewModeForContext(GetCamActiveViewModeContext()) == 4)
+            Wait(0)
+        end
+    end)
+end
+
+function Adjustments:StopPoint()
+    RequestTaskMoveNetworkStateTransition(ESX.PlayerData.ped, 'Stop')
+    ClearPedSecondaryTask(ESX.PlayerData.ped)
+    if not ESX.PlayerData.vehicle then
+        SetPedCurrentWeaponVisible(ESX.PlayerData.ped, true, true, true, true)
+    end
+end
+
+function Adjustments:Point()
+    if not Config.Pointing.Enable then
```

Comment:
`Config.Pointing.Enable` does not exist. You meant `Config.Pointing.enable`. I am going to bed, lmk when you actually tested this PR x)